src/encoded/static/libs/formatMeasurement.js

```diff
@@ -0,0 +1,28 @@
+
+/**
+* Takes a magnitude and unit (singular form) and returns the measurement
+* with the proper unit in singular or plural, depending on magnitude.
+* Invalid measurements are turned as- Unknown
+* @param {number} magnitude - Measured value
+* @param {string} unit - Measurement system in singular form
+*/
+const formatMeasurement = (magnitude, unit) => {
+    // parseFloat used because parseInt only keeps integer
+    const magnitudeAsFloat = parseFloat(magnitude);
+
+    if (Number.isNaN(magnitudeAsFloat)) {
+        return 'Unknown';
+    }
+
+    if (magnitudeAsFloat === 0) {
+        return '0';
+    }
```

Comment:
So a 0 magnitude should never show any units? I didn’t see that in the ticket and just wanted to be sure.